src/pattern_matching/pssm/dnamotif.rs

```diff
@@ -0,0 +1,244 @@
+// Copyright 2018 Kieran Hervold
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use super::*;
+use ndarray::prelude::Array2;
+use std::f32;
+use std::f32::{INFINITY, NEG_INFINITY};
+
+/// Position-specific scoring matrix for DNA sequences
+#[derive(Clone, Debug, PartialEq)]
+pub struct DNAMotif {
+    /// matrix holding weights at each position, indexed by [position, base]
+    pub scores: Array2<f32>,
+    /// sum of "worst" base at each position
+    pub min_score: f32,
+    /// sum of "best" base at each position
+    pub max_score: f32,
+}
+
+impl DNAMotif {
+    /// Returns a Motif representing the sequences provided.
+    /// # Arguments
+    /// * `seqs` - sequences incorportated into motif
+    /// * `pseudos` - array slice with a pseudocount for each monomer;
+    ///    defaults to DEF_PSEUDO for all if None is supplied
```

Comment:
Say ```pssm::DEF_PSEUDO``` here and make it public. Otherwise, nobody would be able to see what is the default.